package.json

```diff
@@ -145,7 +145,7 @@
     "videojs-transcript-ac": "^0.8.8",
     "videojs-youtube": "^2.6.1",
     "walk": "^2.3.14",
-    "webpack": "^4.44.0",
+    "webpack": "^5.73.0",
```

Comment:
Why are we upgrading webpack in this PR?  Did you base your initial branch off of https://github.com/mitodl/ocw-hugo-themes/pull/787?  This PR should be based off of `main` and should only make the change that you made in `.github/workflows/deploy.yml`.